apps/site/src/_pages/docs/2_working-with-types.mdx

```diff
@@ -0,0 +1,67 @@
+# Working with types
+
+## Introduction
+
+When using the Block Protocol's [graph module](/docs/spec/graph-module), data is shared between applications and blocks
+in the form of 'entities', and each entity is assigned to at least one type, known as an 'entity type'.
+
+This type describes what properties an entity has, what type of value each is expected to be, and the relationship
+between the entity in question and other entities.
+
+Relationships between entities are known as 'links' and they too are entities, which may have properties and relations to other entities.
+
+The system by which types are defined is known as the 'type system'.
+
+## The type system
+
+The Block Protocol type system is designed so that types are composed of re-usable, sharable parts, which can be
+referenced by other users and by other types.
+
+The type system consists of these primitive parts:
+
+1.  **Entity types**: define what properties an entity may have (which _property types_), and what relationships (links) it may have with other entities
+2.  **Property types**: define properties that can be referenced by entity types, and their expected values, which may be a reference to
+    _data types_ or objects which are made up of other property types
+3.  **Data types**: primitive data types, e.g. text, numbers, booleans.
+
+## Creating types
+
+The easiest way to get started is to visit [your dashboard](/dashboard) and clicking 'create an entity type'.
```

Comment:
I think this should be "to visit" and "click", or by "by visiting" and "clicking"

Reply:
Done in https://github.com/blockprotocol/blockprotocol/commit/a99d4f2457ac5c73af8b6b4d164455591eaf0b0a